### PR TITLE
improve docs-velero

### PR DESCRIPTION
**Pull request by cutezhangq**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it:**
Improve working-with-velero

**Which issue(s) this PR fixes: **
Fixes #

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**
NONE

### CODE REVIEW

File: docs/proposals/caching/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Goals:
 
 We propose a new commponent named `karmada-search`, it provides a new api group called `search.karmada.io`, the reason why select the `search` word is because it's more to [OpenSearch](https://opensearch.org) which is a community-driven, Apache 2.0-licensed open source search and analytics suite.
 
-The `karmada-search` component currently supports two types of backend stores, namely `cache` and `opensearch`. It uses the `cache` type as the default backend store of the caching layer for Karamda.
+The `karmada-search` component currently supports two types of backend stores, namely `cache` and `opensearch`. It uses the `cache` type as the default backend store of the caching layer for Karmada.
```
Comment (RainbowMango) [inline docs/proposals/caching/README.md:54]: nice finding.

File: docs/working-with-velero.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
 # Integrate Velero to back up and restore Karmada resources
 
-[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore 
-your Kubernetes cluster resources and persistent volumes. You can run Velero with a public
-cloud platform or on-premises.
+[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. You can run Velero with a public cloud platform or on-premises.
```
Comment (RainbowMango) [inline docs/working-with-velero.md:3]: I don't think we need to put the content in one line.

File: docs/working-with-velero.md [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
 - Replicate your production cluster to development and testing clusters.
 
-This document gives an example to demonstrate how to use the `Velero` to back up and restore Kubernetes cluster resources
-and persistent volumes. Following example backups resources in cluster `member1`, and then restores those to cluster `member2`.
+This document provides an example to demonstrate how to use `Velero` to back up and restore Kubernetes cluster resources and persistent volumes.
+For example, backups resources in cluster `member1`, and then restore those to cluster `member2`.
 
-## Start up Karamda clusters
-You just need to clone Karamda repo, and run the following script in Karamda directory.
+## Start up Karmada clusters
```
Comment (RainbowMango) [inline docs/working-with-velero.md:14]: nice finding.

File: docs/working-with-velero.md [hunk 4]
```diff
@@ -104,27 +97,27 @@ Velero consists of two components:
...
-   And then you will find nginx is deployed successfully.
+   Then, you will find nginx is deployed successfully.
    ```shell
    # kubectl get deployment.apps
    NAME    READY   UP-TO-DATE   AVAILABLE   AGE
    nginx   2/2     2            2           17s
    ```
 
-### Back up and restore kubernetes resources independent
+### Back up and restore Kubernetes resources independent
```
Comment (RainbowMango) [inline docs/working-with-velero.md:112]: +1

File: docs/working-with-velero.md (hunk 2 above)
Comment (cutezhangq) [reply docs/working-with-velero.md]: OK, I have modified it.

File: docs/working-with-velero.md [hunk 5]
```diff
@@ -1,25 +1,25 @@
 # Integrate Velero to back up and restore Karmada resources
 
-[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore 
-your Kubernetes cluster resources and persistent volumes. You can run Velero with a public
-cloud platform or on-premises.
+- [Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore.
+- Your Kubernetes cluster resources and persistent volumes. You can run Velero with a public.
+- Cloud platform or on-premises.
```
Comment (RainbowMango) [inline docs/working-with-velero.md:5]: Why?

**Comment by RainbowMango** [comment]
Hi @cutezhangq , I appreciated your contribution, but please explain the reason why we need that.

File: docs/working-with-velero.md (hunk 5 above)
Comment (cutezhangq) [reply docs/working-with-velero.md]: Sorry, my branch is a little messy. I'll close this submission first and resubmit later. I have read this sentence carefully and will not make any additional modifications.

**Pull request closed without merging**